EIPS/eip-1.md

```diff
@@ -42,19 +42,23 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
+An automated process will check all pull requests; if the request passes the format checks and contains only additions or edits to drafts owned by the submitter, it will be automatically merged. No editorial control is exercised over the content of an EIP draft.
 
-For an EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+Standards track EIPs of type 'Core' are updated from 'Draft' to 'Accepted' status when a core devs meeting has accepted the draft and the participants have announced their intention to implement the EIP for inclusion in a future hard fork. At that point, an EIP number is assigned by the editors and the EIP is moved to the EIPS directory and renamed accordingly. Once a Core EIP has been implemented in at least three clients, and all pass a common set of test suites, the status of the EIP is updated to 'Final'. An EIP can also be assigned status “Deferred”. The EIP author or editor can assign the EIP this status when no progress is being made on the EIP.
 
-Once an EIP has been accepted, the implementations must be completed. When the implementation is complete and accepted by the community, the status will be changed to “Final”.
+Standards track, non-core EIPs remain in Draft status until at least one implementation has been completed and referenced in the Implementations section of the draft. At that point, the draft's author may apply to the EIP editor, by way of a pull request, to assign a number to the EIP and update its status to 'Final', moving it to the EIPs directory and renaming it accordingly.
 
-An EIP can also be assigned status “Deferred”. The EIP author or editor can assign the EIP this status when no progress is being made on the EIP. Once an EIP is deferred, the EIP editor can re-assign it to draft status.
+Non-standards track EIPs may be transitioned to 'Final' status when the author requests it, and at the discretion of the editors.
+
+For an EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly. The EIP editor will not unreasonably deny an EIP. Reasons for denying EIP status include duplication of effort, being technically unsound, not providing proper motivation, or failing to address backwards compatibility.
+
+Once an EIP is marked Final, edits are not permitted unless they correct errata; no substantive changes may be made. Anyone wishing to amend a Final EIP must submit a new EIP instead.
 
 An EIP can also be “Rejected”. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
 
```

Comment:
Who Rejects an EIP.  The core devs?  And who Withdraws an EIP?  The author?  This is the point at which I've suggested an EIP either goes to Deferred, Withdrawn, or Ready.  Ready means the editors and authors agree the Draft is ready for consideration by the core devs.  It's also at this point that the Fellowships consensus becomes relevant.  Seems we editors and the autor can choose whether to take that consensus as just input, or to respect it as a matter of policy.

Reply:
Good question RE rejection; I don't know.

I don't think Ready is necessary as a formal status; it's ready when the author asks it to be added to an All Core Devs agenda.